examples/local-dev-values.yaml

```diff
@@ -0,0 +1,34 @@
+pachd:
+  storage:
+    backend: LOCAL
+  image:
+    tag: local
+  resources:
+    limits:
+      cpu: 250m
+      memory: 512M
+    requests:
+      cpu: 250m
+      memory: 512M
+  service: 
+    type: NodePort
+  metrics:
+    enabled: false
+  blockCacheBytes: 256M
+  authenticationDisabledForTesting: true
+  exposeObjectAPI: true
+  exposeDockerSocket: true
+
+
+dash:
+  # resources: {} Doesn't work
```

Comment:
I think that this will work now … there is a conditional guard in place. But this doesn’t block merging.

Reply:
thanks! will get rid of that note